nginx.conf

```diff
@@ -94,6 +94,7 @@
     # Grafana
     rewrite ^/$ $scheme://$http_host/graph/;
     rewrite ^/graph$ /graph/;
+    rewrite ^/explore(.*)$ /graph/explore$1 permanent;
```

Comment:
Could you move the line into the section below?
![Screenshot_20200914_183811](https://user-images.githubusercontent.com/25184101/93107006-b4beb080-f6b9-11ea-8601-b138be1b9206.png)

Reply:
Yes I can, but then the comment will mislead saying that this line can be removed after fixing that problem, which is not the case for this. Should I still move it?

Reply:
Do we have upstream issue for that?

Reply:
No. But I don't think the problem is upstream since they don't use /graph.

Reply:
They provide the root_url parameter, and they should honor it. Please check the linked issue, then find or create n issue for this location, then add a link there.

Reply:
You are right. Didn't find an issue so created one [here](https://github.com/grafana/grafana/issues/27588) and added the link in a comment.